### ISSUE TITLE
Open details element by default for citytod and pudo line charts

### ISSUE THREAD

**Issue by KatiRG** [CityofToronto/bdit_cot-vfh#42]
Details element cannot be toggled open with tab key in Edge so it must be set open always
